### PR TITLE
[FLINK-6439] Fix close OutputStream && InputStream in OperatorSnapshotUtil

### PR DESCRIPTION
**Pull request by zjureel**
…tUtil

Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [ ] General
  - The pull request references the related JIRA issue ("[FLINK-XXX] Jira title text")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [ ] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

**Comment by zjureel** [comment]
@zentol I have fix the issue 6439 in this PR, could you please review the code here when you are free. Thaks :)

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/util/OperatorSnapshotUtil.java [hunk 1]
```diff
@@ -46,111 +46,113 @@ public static String getResourceFilename(String filename) {
...
-
-		SavepointV1Serializer.serializeStreamStateHandle(state.getLegacyOperatorState(), dos);
-
-		Collection<OperatorStateHandle> rawOperatorState = state.getRawOperatorState();
-		if (rawOperatorState != null) {
-			dos.writeInt(rawOperatorState.size());
-			for (OperatorStateHandle operatorStateHandle : rawOperatorState) {
-				SavepointV1Serializer.serializeOperatorStateHandle(operatorStateHandle, dos);
+		
+		try(DataOutputStream dos = new DataOutputStream(out)) {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/util/OperatorSnapshotUtil.java]: missing space after try

File: flink-streaming-java/src/test/java/org/apache/flink/streaming/util/OperatorSnapshotUtil.java [hunk 2]
```diff
@@ -46,111 +46,113 @@ public static String getResourceFilename(String filename) {
...
-		out.close();
+			dos.flush();
+		}
 	}
 
 	public static OperatorStateHandles readStateHandle(String path) throws IOException, ClassNotFoundException {
 		FileInputStream in = new FileInputStream(path);
-		DataInputStream dis = new DataInputStream(in);
-		int index = dis.readInt();
+		try(DataInputStream dis = new DataInputStream(in)) {
```
Comment (zentol) [inline flink-streaming-java/src/test/java/org/apache/flink/streaming/util/OperatorSnapshotUtil.java]: missing space after try.

**Comment by zjureel** [comment]
@zentol Thank you for your reply, I have update the code and add space after `try`

**Comment by zentol** [comment]
merging.

**Pull request closed without merging**